src/main/java/kr/dgucaps/caps/domain/wiki/entity/Wiki.java

```diff
@@ -31,15 +31,19 @@ public class Wiki extends BaseTimeEntity {
     @Column(nullable = false, columnDefinition = "TEXT")
     private String content;
 
+    @Column(nullable = false)
+    private String jamo;
+
     @Column(nullable = false)
     @ColumnDefault("false")
     private boolean isDeleted;
 
```

Comment:
이거 쓰는게 어떨지 고민 한번 해보세요!
```suggestion
@PrePersist
public void initJamo() {
    this.jamo = JamoUtil.convertToJamo(this.title);
}
```